tsconfig.json

```diff
@@ -19,5 +19,10 @@
   },
   "include": ["./src"],
   "types": ["node", "jest"],
-  "exclude": ["src/**/*.test.ts", "markdown.config.js"]
+  "exclude": ["src/**/*.test.ts", "markdown.config.js"],
+  "typedocOptions": {
+    "entryPoints": ["src/index.ts"],
+    "json": "scripts/markdown-magic/typedoc.out.json",
+    "plugin": ["typedoc-plugin-missing-exports"]
+  }
```

Comment:
`pretty` option is enabled by default, could be expressly defined though

Reply:
that's great!